### ISSUE TITLE
Casting For-Loop Iterator to Match Address Width in Decoding Logic

### ISSUE THREAD

**Issue by leolitenstorrent** [SystemRDL/PeakRDL-regblock#92]
Dear PeakRDL Development Team,

```
    //--------------------------------------------------------------------------
    // Address Decode
    //--------------------------------------------------------------------------
    typedef struct {
        // Code not shown
    } decoded_reg_strb_t;
    // Code not shown

    always_comb begin
        // Code not shown
        for(int i0=0; i0<32; i0++) begin
            decoded_reg_strb.REGISTER_ARRAY[i0] = cpuif_req_masked & (cpuif_addr == 11'hf8 + i0*11'h8);
        end
    end
```

In the SV code generated above for address decoding, the variable `i0` has bitwidth 32 whereas the address has bit width 11 (in this case). This results in the Lint warning of "Unequal length in comparison operator". May the team consider resolving this? For example, `i0*11'h8` can be changed to `11'(i0)*11'h8`.

Thank you!

**Comment by amykyta3** [comment]
Easy enough to add.
Width cast will be included in the next upcoming release.

**Comment by amykyta3** [comment]
Implemented in v0.23.0
